api/Makefile

```diff
@@ -1,6 +1,10 @@
+SHELL:=/bin/bash
 L0_VERSION?=$(shell git describe --tags)
 API_DOCKER_IMAGE=quintilesims/l0-api:$(L0_VERSION)
 
+deps:
+	@docker login -u $(DOCKERHUB_USERNAME) -p $(DOCKERHUB_PASSWORD)
```

Comment:
No. The current workflow here is not running in a container, anyway. Having it in `deps` here will allow this to work in either a docker or non-docker context